src/Components/Navigation.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+function Navbar() {
+  return (
+    <nav>
+      <div className="right">
+        <Link to="/">
+          <h2>Book-State</h2>
+        </Link>
+
+        <Link to="/categories">
+          <h2>Book-List</h2>
```

Comment:
- [ ] You have done great changing the component to `Category` as the previous reviewer suggested, but you haven't updated the link text. I'll suggest you do so. :smile: :wave: